### PR TITLE
Update translation files

### PR DESCRIPTION
**Pull request by cea2aj**
Update translations for all languages

J=none
TEST=manual

Used "msgcmp" to confirm all the translations are present

### CODE REVIEW

File: conf/i18n/translations/it.po [hunk 1]
```diff
@@ -170,7 +176,7 @@ msgstr "inviare feedback"
 
 #: src/ui/templates/results/alternativeverticals.hbs:5
 msgid "Showing <em class=\"yxt-AlternativeVerticals-noResultsInfo--emphasized\">all [[currentVerticalLabel]]</em> instead."
-msgstr "Mostrando <em class=\"yxt-AlternativeVerticals-noResultsInfo--emphasized\">invece che [[currentVerticalLabel]] </em>."
+msgstr "Mostrando <em class=\"yxt-AlternativeVerticals-noResultsInfo--emphasized\">tutte le [[currentVerticalLabel]] disponibili.</em>."
```
Comment (rosiegrant) [inline conf/i18n/translations/it.po]: extra period inside the <em> tag

Comment (cea2aj) [reply conf/i18n/translations/it.po]: fixed!

File: conf/i18n/translations/de.po [hunk 2]
```diff
@@ -250,6 +312,31 @@ msgctxt "Displays more options"
...
+
+#: src/ui/templates/results/resultsheader.hbs:21
+msgctxt "Example: 1-10 of 50"
+msgid "<span class=\"yxt-ResultsHeader-resultsCountStart\">[[start]]</span> <span class=\"yxt-ResultsHeader-resultsCountDash\">-</span> <span class=\"yxt-ResultsHeader-resultsCountEnd\">[[end]]</span> <span class=\"yxt-ResultsHeader-resultsCountOf\">of</span> <span class=\"yxt-ResultsHeader-resultsCountTotal\">[[resultsCount]]</span>"
+msgstr "<span class=\"yxt-ResultsHeader-resultsCountStart\">[[start]]</span> <span class=\"yxt-ResultsHeader-resultsCountDash\">-</span> <span class=\"yxt-ResultsHeader-resultsCountEnd\">[[end]]</span> <span class=\"yxt-ResultsHeader-resultsCountOf\">von</span> <span class=\"yxt-ResultsHeader-resultsCountTotal\">[[resultsCount]]</span>"
+
+#: src/ui/templates/results/verticalresultscount.hbs:5
+msgctxt "Example: 1-10 of 50"
+msgid ""
+"<span class=\"yxt-VerticalResultsCount-start\">[[start]]</span>\n"
```
Comment (tmeyer2115) [inline conf/i18n/translations/de.po:328]: Does it matter that this `msgid` and `msgstr` have an empty `""` and then a newline before the associated value?

Comment (cea2aj) [reply conf/i18n/translations/de.po:328]: Nope! Oliver and I learned that this is possible from the PO file spec and we also tested the translation in a site.

**Review by tmeyer2115 — APPROVED** [review]

**Pull request merged**